mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -614,7 +614,12 @@ var ServicesListGa = mapOf(
     "securitycentermanagement" to mapOf(
         "name" to "securitycentermanagement",
         "displayName" to "Securitycentermanagement",
-        "path" to "./google/services/securitycentermanagement"
+        "path" to "./google-beta/services/securitycentermanagement"
+    ),
+    "securitycenterv2" to mapOf(
+        "name" to "securitycenterv2",
+        "displayName" to "securitycenterv2",
+        "path" to "./google-beta/services/securitycenterv2"
```

Comment:
I believe this should be changed to `./google/services/securitycenterv2`

Reply:
Still unaddressed

Reply:
addressed